config/dev.webpack.config.js

```diff
@@ -28,8 +30,10 @@ plugins.push(
     })
 );
 
+webpackConfig.devServer.client.overlay = false;
+
 module.exports = (env) => {
     env && env.analyze === 'true' && plugins.push(new BundleAnalyzerPlugin());
 
-    return { ...webpackConfig, plugins };
+    return { ...webpackConfig, plugins: plugins.filter((plugin) => plugin.constructor.name !== 'HtmlWebpackPlugin') };
```

Comment:
HTML plugin was also changed in the latest version. The JS injection was disabled.

Reply:
The problem here was with ESI, for some reason when running locally it just broke and wouldn't show any content. Once the plugin was disabled it started to run.